plugins/package-managers/sbt/src/funTest/assets/projects/target-scala-2.12-projects_2.12-0.1.0-SNAPSHOT.pom

```diff
@@ -0,0 +1,20 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
The files committed here and below seem like temporary left-overs.

Reply:
indeed, using git add -A is not really a good idea.